README.md

```diff
@@ -1,4 +1,60 @@
+![Logo](readmeImages/fsLogo.png "FullStory Logo")
+
 # ios-shoppe-demo
 iOS Client for the example "Shoppe" app
 
-_Stay tuned!_
+The iOS Shoppe Demo is a reference application that provides tips and tricks to using [FullStory](https://www.fullstory.com/) on [Native Mobile](https://www.fullstory.com/mobile-apps/) iOS.
+
+## Getting started
+
+To apply the FullStory iOS Plugin, you'll need [Xcode](https://developer.apple.com/xcode). You'll then download or clone this repo to your desired directory.
+
+To open the project, use "File > Open" in Xcode and select either "swift" or "Objective-C"(coming soon) folder.
+
+## Adding FullStory to the app
+
+If you have [signed up](https://www.fullstory.com/plans/) for FullStory, add the following to your `Info.plist`:
+```
+<key>FSOrgId</key>
+  <string>YOUR_ORG_HERE</string>
+<key>FSHost</key>
+  <string>fullstory.com</string>
+```
+
+Your orgId can be found on the _Settings_ page once you are logged in to FullStory.  It'll be next to the `window['_fs_org']` variable in the snippet.  For more information, see [Install your recording snippet](https://help.fullstory.com/hc/en-us/articles/360020828233#Install).
+
+For successful recording, please do not change the `server` value. It should always be `https://fullstory.com`.
+
+Sync gradle after the changes and your app should be fully instrumented with your org information.
+
+## Using the app
+
+The Shoppe is a super simple e-commerce application. Build and run the app on your emulator, you can:
+
+- Browse a list of products int the _Market_.
+- Use the **Add to Cart** button to add products to your shopping cart.
+- Go to your _Shopping Cart_ by clicking on the cart icon on the top right corner.
+- Review your cart and then click the **Checkout** button.
+- Fill out the form on the _Checkout_ view and click **Purchase**. The app persists locally all the information from this view. Please do **not** use your real information
```

Comment:
Missing period at end.

Reply:
Does the app also have exclusion rules if the end user did try to enter sensitive information?  If so, we might want to re-iterate that point on what PII is/isn't recorded in sessions.

Reply:
Interesting, well at the moment I have not built this out but I did not plan on recording this info, I will bring it up again once the checkout flow is complete.